doc/source/user_guide/machine_learning_pysteps.rst

```diff
@@ -0,0 +1,103 @@
+.. _machine_learning_pysteps:
+
+Machine learning vs pysteps
+===========================
+How to correctly compare the accuracy of machine learning against pysteps?
+
+Before starting, you need to ask yourself these two questions:
+
+#. Do you only want to minimize prediction errors?
+#. Do you also want to represent the prediction uncertainty? 
+
+To achieve objective 1, it is sufficient to produce a single deterministic nowcast that filters out the unpredictable small-scale precipitation features.
+However, this will create a nowcast that will become increasingly smooth over time.
+
+To achieve objective 2, you need to produce a probabilistic or an ensemble nowcast (several ensemble members and realizations).
+
+In weather forecasting (and nowcasting) we usually want to achieve both goals because it is impossible to predict the evolution of a chaotic system with 100% accuracy, especially space-time precipitation fields and thunderstorms! 
+
+Machine learning and pysteps offer several methods to produce both deterministic and probabilistic nowcasts. 
+Therefore, if you want to compare machine learning-based nowcasts to pysteps you need to select the right method.
+
+1. Deterministic machine learning vs pysteps
+--------------------------------------------
+
+Deterministic nowcasts can be divided into:
+
+a. Variance-preserving nowcasts, such as extrapolation nowcasts by Eulerian and Lagrangian persistence.
+b. Error-minimization nowcasts, such as machine learning-based or ensemble mean nowcasts.
+
+**Very important**: these two types of deterministic nowcasts are not directly comparable because they have a different variance! 
+This is best explained by the decomposition of the mean squared error (MSE):
+
+:math:`MSE = bias^2 + Var`
+
+All deterministic machine learning algorithms that minimize the MSE (or a variation) will also inevitably minimize the variance of nowcast fields.
+This is a natural attempt to filter out the unpredictable precipitation features, which would otherwise increase the variance (and the MSE).
+The same principle holds for convolutional and/or deep neural network architectures, which also produce smooth nowcasts.
+
+Therefore, you should never directly compare an error-minimization machine learning nowcast to a variance-preserving radar extrapolation, as produced by the module pysteps.nowcasts.extrapolation. Instead, you should use the pysteps ensemble mean.
```

Comment:
```suggestion
Therefore, it is better to avoid directly comparing an error-minimization machine learning nowcast to a variance-preserving radar extrapolation, as produced by the module pysteps.nowcasts.extrapolation. Instead, you should use the pysteps ensemble mean.
```

Reply:
Also, do you think it could be better to link the modules you list here and below directly to the corresponding documentation page?